test/controllers/ATSMainControllerTest.scala

```diff
@@ -18,45 +18,37 @@ package controllers
 
 import config.AppFormPartialRetriever
 import org.jsoup.Jsoup
+import org.mockito.Matchers
 import org.mockito.Mockito._
-import org.mockito.Matchers._
-import org.scalatest.mock.MockitoSugar
-import play.api.mvc.Request
-import play.api.test.Helpers._
+import org.scalatest.mockito.MockitoSugar
 import play.api.test.FakeRequest
+import play.api.test.Helpers._
 import services._
 import uk.gov.hmrc.play.frontend.auth.{AuthContext => User}
+import uk.gov.hmrc.play.partials.FormPartialRetriever
 import uk.gov.hmrc.play.test.UnitSpec
 import utils.AuthorityUtils
-
-import scala.concurrent.Future
 import utils.TestConstants._
-import uk.gov.hmrc.http.HeaderCarrier
-import uk.gov.hmrc.play.partials.FormPartialRetriever
+import scala.concurrent.Future
 
 class ATSMainControllerTest extends UnitSpec with FakeTaxsPlayApplication with MockitoSugar {
 
   val user = User(AuthorityUtils.saAuthority(testOid, testUtr))
-  val request = FakeRequest()
-
+  val taxYear  =2015
   val baseModel = SummaryControllerTest.baseModel
+  val request = FakeRequest("Get",s"?taxYear=$taxYear")
```

Comment:
Please change any "Get" to "GET" in your fake requests (More below)

Reply:
Changed to GET